### PR TITLE
Adding Julian day numbers to Make date dialogue

### LINKED ISSUE #4974: Adding Julian day numbers to Make date dialogue
**Is your feature request related to a problem? Please describe.**
This is another possible option when there is a single column for the day number and you specify the origin.  There are currently 3 options, namely Excel, Gregorian and Specify.  The data from CHIRPS, etc (from IRI data library have the date column (via NetCDF) as Julian day number.  This is when the origin is 1 January 4713BC.  So 1st January 1981 is day number 2444606.   

**Describe the solution you'd like**
With the current dialogue I first subtracted 2436204.5.  This makes the 1st January 1958 as day 0.  Then I used the existing dialogue with that as the start.  Worked fine.  
That's not how I would program it mind!  It says (Wikipedia) that floor (JD − 1721424.5) = Gregorian day number, which we have.

I hope this might be a quick addition - who would like to take it?

### PR DESCRIPTION
**Pull request by Wycklife**
This fixes #4974 
@africanmathsinitiative/developers. This is ready for review

### CODE REVIEW

**Comment by dannyparsons** [comment]
Thanks @Wycklife. 
@rdstern do you have an example dataset with Julian dates so that whoever tests this has a good example? Could you provide your NETcdf file where this came up or provide a csv of the data frame that R-Instat creates from it?

**Comment by rdstern** [comment]
I am having a problem reading any data just now with the netcdf to check.  I think these files could be tried, but I need to be able to read them first.

[tamsat_data.zip](https://github.com/africanmathsinitiative/R-Instat/files/2871950/tamsat_data.zip)

These give me an error when reading as netcdf

**Comment by dannyparsons** [comment]
@Wycklife and @africanmathsinitiative/developers a good test for this is to import one of the NETcdf files in the zip file Roger attached above. You will get a `T` column for time as a number in Julian days. See if this converts correctly to a date. The data starts on something like 1/1/1981.

**Comment by Patowhiz** [comment]
Testing this

**Comment by Patowhiz** [comment]
I used the chirps_data to test this, the dates convert well. I verified this using the [online Julian Date Converter](http://www.onlineconversion.com/julian_date.htm).

Though trivial you could rearrange the tab index order a bit.

**Comment by dannyparsons** [comment]
Yes I think "Specify" should stay at the bottom and the others should be in alphabetical order. Could you also add `...` to Specify to identify that it brings up further options?

**Comment by Muthenya** [comment]
@Wycklife were you able to make these proposed changes by both Patrick and Danny?

**Comment by Wycklife** [comment]
@Muthenya Could you review this please

**Comment by dannyparsons** [comment]
I see that @Muthenya has been given a lot to review! How about someone else? @Ivanluv ?

**Comment by maxwellfundi** [comment]
@dannyparsons  I have retested this and it works well! Can be merged in

File: instat/dlgMakeDate.resx [hunk 1]
```diff
@@ -1778,17 +2246,14 @@ Examples:
...
   </data>
   <data name="$this.ImeMode" type="System.Windows.Forms.ImeMode, System.Windows.Forms">
     <value>NoControl</value>
   </data>
   <data name="$this.StartPosition" type="System.Windows.Forms.FormStartPosition, System.Windows.Forms">
     <value>CenterScreen</value>
   </data>
-  <data name="$this.Text" xml:space="preserve">
-    <value>Make Date</value>
-  </data>
```
Comment (maxwellfundi) [inline instat/dlgMakeDate.resx]: @Wycklife  Seems to have been removed here.. Please get it back.

Comment (dannyparsons) [reply instat/dlgMakeDate.resx]: @maxwellfundi is it something you can correct quickly so it can be merged?

**Pull request merged**